### PR TITLE
Put browser-compat info in front-runner for api/s*

### PR DESCRIPTION
**Pull request by teoli2003**
> What was wrong/why is this fix needed? (quick summary only)

We want to have browser-compat in front-runner: this allows us to write {{Compat}} without parameter, and in the future, {{Spec}} (and likely more).

This PR covers api/s* for the case without problem: 1 macro Compat in the page and its parameter matching the slug. Other cases (no Compat macros on a page, multiple Compat macros, macro not matching the slug, will be done at a later stage).

> MDN URL of the main page changed

539 files in api/* 

> Issue number (if there is an associated issue)

openwebdocs/project#36

> Anything else that could help us review it

### CODE REVIEW

File: files/en-us/web/api/subtlecrypto/importkey/index.html [hunk 1]
```diff
@@ -369,7 +369,8 @@ <h3 id="SubjectPublicKeyInfo_import">SubjectPublicKeyInfo import</h3>
   function importRsaKey(pem) {
     // fetch the part of the PEM string between header and footer
     const pemHeader = "-----BEGIN PUBLIC KEY-----";
-    const pemFooter = "-----END PUBLIC KEY-----";
+    const pemFooter = "-----END PUBLIC KEY--browser-compat: api.SubtleCrypto.importKey
```
Comment (Elchi3) [inline files/en-us/web/api/subtlecrypto/importkey/index.html:372]: Something is going on here.

File: files/en-us/web/api/subtlecrypto/exportkey/index.html [hunk 2]
```diff
@@ -216,7 +216,8 @@ <h3 class="highlight-spanned" id="SubjectPublicKeyInfo_export"><span
   );
   const exportedAsString = ab2str(exported);
   const exportedAsBase64 = window.btoa(exportedAsString);
-  const pemExported = `-----BEGIN PUBLIC KEY-----\n${exportedAsBase64}\n-----END PUBLIC KEY-----`;
+  const pemExported = `-----BEGIN PUBLIC KEY-----\n${exportedAsBase64}\n-----END PUBLIC KEY--browser-compat: api.SubtleCrypto.exportKey
```
Comment (Elchi3) [inline files/en-us/web/api/subtlecrypto/exportkey/index.html:219]: Same here.

**Review by Elchi3 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/subtlecrypto/exportkey/index.html (hunk 2 above)
Comment (Elchi3) [reply files/en-us/web/api/subtlecrypto/exportkey/index.html:219]: btw I haven't checked if it happens elsewhere. The Crypto API pages might have more of this.

File: files/en-us/web/api/subtlecrypto/importkey/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/subtlecrypto/importkey/index.html:372]: Fixed (lazy regexp).

File: files/en-us/web/api/subtlecrypto/exportkey/index.html (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/api/subtlecrypto/exportkey/index.html:219]: Fixed (lazy crypto).

I checked the whole Web Crypto API to be sure: these were the only two cases with cryptographic keys (that contains the --- trigerring the problem).

**Review by Elchi3 — APPROVED** [review]
Thank you! 👍

**Pull request merged**